### ISSUE TITLE
Unable to save edits to getContentsBlock in Enhanced list view

### ISSUE THREAD

**Issue by jgfoster** [dolphinsmalltalk/Dolphin#955]
**Describe the bug**
Unable to save code edits in View Composer aspect editor.

**To Reproduce**
1. Create a subclass of Shell
2. Create a Default View
3. Add a ListPresenter.Enhanced list view
4. Expand columnsList, first column
5. Double-click getContentsBlock
6. Enter a block expression: [:each | each first]
7. Save with green check button
8. Content stays as nil

**Expected behavior**
Code saved.

**Screenshots**
![Screen Shot 2020-06-18 at 10 37 35 PM](https://user-images.githubusercontent.com/1577872/85100179-56fef300-b1b4-11ea-9fd3-7e1e24221b2a.png)
**Please complete the following information):**
 - OS: Windows 10

**Additional context**
I've verified the problem back to Sept 11, 2019 (the first build to support VS2019). 

It has been a while since I've edited an Enhanced list view, but I think I'm doing the right things.

**Comment by jgfoster** [comment]
I wonder if this is related to the fact that ValueWorkspace "is a somewhat unusual MVP triad; the ValueWorkspace model is not connected directly to the view. Instead, the view is redirected (via #viewModel) to another model that holds the textual rendition of the actual model." 

In `AspectEditorDialog>>apply`, the following line changes things so that the text entered becomes a String instead of leaving things as `nil`:

```
    "subjectModel value: aspectPresenterInDialog model value."
     subjectModel value: aspectPresenterInDialog viewModel value.
```

This is an improvement, but it eventually needs to be a block of code, not just a string.

**Comment by blairmcg** [comment]
This bug has been around (and annoying me) for a while, I just haven't got around to logging it. Part of the reason for my laziness is that there is a workaround. If you first "Accept" in the workspace (using the menu) before pressing the OK tick, it should work. The workaround does at least allow the edit to be completed, although most of the time I end up doing it more than once after losing the change the first time because I forget that this doesn't work properly.

I'm not sure whether this ever worked correctly with the revised in-place editing in the VC that was put in for 7.0, or whether it was something that got broken later. There was a related problem that I think perhaps you might have logged James, and which I fixed but which may have caused this problem.

**Comment by blairmcg** [comment]
Actually it was #575, and really very similar, so it must have regressed.

**Comment by blairmcg** [comment]
Fixed by #956
